SPECS-EXTENDED/hdf/hdf.spec

```diff
@@ -0,0 +1,578 @@
+# No more Java on i686
+%ifarch %{java_arches}
+%bcond_without java
+%else
+%bcond_with java
+%endif
+Summary:        A general purpose library and file format for storing scientific data
+Name:           hdf
+Version:        4.2.15
```

Comment:
I think we have `hdf5`, does not suffice for our usecase ?
Reference: https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/hdf5/hdf5.spec#L14

Reply:
No @mfrw  
Reference : https://gdal.org/development/building_from_source.html#hdf4